tests/perf_v2/tasks/detection.py

```diff
@@ -33,28 +33,96 @@
     ModelInfo(task=TASK_TYPE.value, name="yolox_x", category="other"),
 ]
 
-DATASET_TEST_CASES = [
-    DatasetInfo(
-        name="pothole_small_1",
-        path=Path("detection/pothole_small/1"),
-        group="small",
-        extra_overrides={
-            "test": {
-                "metric": FMeasureCallable,
+DATASET_TEST_CASES = (
+    [
+        DatasetInfo(
+            name=f"pothole_tiny_{idx}",
+            path=Path("detection/pothole_coco_tiny") / f"{idx}",
+            group="tiny",
+            extra_overrides={
+                "test": {
+                    "metric": FMeasureCallable,
+                },
             },
-        },
-    ),
-    DatasetInfo(
-        name="pothole_small_2",
-        path=Path("detection/pothole_small/2"),
-        group="small",
-        extra_overrides={
-            "test": {
-                "metric": FMeasureCallable,
+        )
+        for idx in (1, 2, 3)
+    ]
+    + [
+        DatasetInfo(
+            name=f"blueberry_tiny_{idx}",
+            path=Path("detection/blueberry_tiny_coco") / f"{idx}",
+            group="tiny",
+            extra_overrides={
+                "test": {
+                    "metric": FMeasureCallable,
+                },
             },
-        },
-    ),
-]
+        )
+        for idx in (1, 2, 3)
+    ]
+    + [
+        DatasetInfo(
+            name="wgisd_small",
+            path=Path("detection/wgisd_merged_coco_small"),
+            group="small",
+            extra_overrides={
+                "test": {
+                    "metric": FMeasureCallable,
+                },
+            },
+        ),
+        DatasetInfo(
+            name="skindetect",
+            path=Path("detection/skindetect-roboflow"),
+            group="small",
+            extra_overrides={
+                "test": {
+                    "metric": FMeasureCallable,
+                },
+            },
+        ),
+        DatasetInfo(
+            name="diopsis",
```

Comment:
I think there are a lot of datasets to test for OD, let's keep 2 tiny, 2 small, 2 medium and 1 large